pages/linux/snap.md

```diff
@@ -0,0 +1,24 @@
+# snap
+
+> Tool for managing the *snap* self-contained softwawre packages.
+> Similar to what *apt* is for *.deb*.
+
+- Look for an app:
```

Comment:
Why not `Find an app`? I see no need to find a synonym for the action.

Reply:
Synonym is just to reinforce in the imaginary of the user the meaning and sense of the action that the command performs. We can leave it at "Find" but that would be just being redundant while wasting a chance for providing a bit more of additional help.